Odd, its dreamwalker from [this](https://github.com/TrinityCore/TrinityCore/commit/9f9507e6a1fd50a5ce643a4096c1712700244a61#commitcomment-26632225) that seems to be failing not this. From [travis](https://travis-ci.org/TrinityCore/TrinityCore/builds/324918053#L1836)

```
/home/travis/build/TrinityCore/TrinityCore/src/server/scripts/Northrend/IcecrownCitadel/boss_valithria_dreamwalker.cpp:867:18: fatal error: 'MovementInform' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
            void MovementInform(uint32 /*type*/, uint32 id)
                 ^
/home/travis/build/TrinityCore/TrinityCore/src/server/game/AI/ScriptedAI/ScriptedCreature.h:169:10: note: overridden virtual function is here
    void MovementInform(uint32 /*type*/, uint32 /*id*/) override { }
         ^
```

--- a/src/server/game/Spells/SpellEffects.cpp
+++ b/src/server/game/Spells/SpellEffects.cpp
@@ -1730,7 +1730,7 @@ void Spell::EffectEnergize(SpellEffIndex effIndex)
 
     Powers power = Powers(m_spellInfo->Effects[effIndex].MiscValue);
 
-    if (unitTarget->GetTypeId() == TYPEID_PLAYER && unitTarget->getPowerType() != power && !(m_spellInfo->SpellFamilyName == SPELLFAMILY_POTION) 
+    if (unitTarget->GetTypeId() == TYPEID_PLAYER && unitTarget->getPowerType() != power && m_spellInfo->SpellFamilyName != SPELLFAMILY_POTION
         && !m_spellInfo->HasAttribute(SPELL_ATTR7_CAN_RESTORE_SECONDARY_POWER))
         return;
 

